src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -13682,12 +13682,6 @@ private QueryContinuationSyntax ParseQueryContinuation()
             return _syntaxFactory.QueryContinuation(@into, name, body);
         }
 
-        [Obsolete("Use IsIncrementalAndFactoryContextMatches")]
-        private new bool IsIncremental
```

Comment:
>IsIncremental [](http://example.com/codeflow?start=25&length=13)

It looks like this property was added intentionally, to catch cases where `IsIncrementalAndFactoryContextMatches` should be used instead.